<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="pcl_nodelet"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/naoqi_driver/camera/depth/camera_info"/>
    <remap from="image_rect" to="/naoqi_driver/camera/depth/image_raw"/>
    <!-- <remap from="points" to="/camera/depth/points"/> -->
  </node>
</launch>